### PR TITLE
budgie-desktop:add SOURCE_DIRECTORY and fix PRE_BUILD

### PR DESCRIPTION
**Pull request by lhw828**
budgie-desktop:add SOURCE_DIRECTORY and fix PRE_BUILD

### CODE REVIEW

**Comment by lhw828** [comment]
Thanks @Florin65 @v4hn @fossfreedom Now, it looks like compiled properly.

File: apps/budgie-desktop/PRE_BUILD [hunk 1]
```diff
@@ -1,6 +1 @@
-default_pre_build &&
-
-sedit 's/verion/version/' meson.build &&
-
-git config --local submodule.subprojects/gvc.url libgnome-volume-control
-git submodule update --init
+default_pre_build && sedit '/meson_post_install/ d' meson.build
```
Comment (v4hn) [inline apps/budgie-desktop/PRE_BUILD]: please add a newline after `&&`

File: apps/budgie-desktop/DETAILS [hunk 2]
```diff
@@ -2,6 +2,7 @@
          VERSION=10.4+git20180830.02.f2dbc215fdb
           SOURCE=${MODULE}_$VERSION.orig.tar.xz
       SOURCE_URL=http://archive.ubuntu.com/ubuntu/pool/universe/b/budgie-desktop/
+      SOURCE_DIRECTORY=$BUILD_DIRECTORY/$MODULE-10.4
```
Comment (v4hn) [inline apps/budgie-desktop/DETAILS]: Please use `${VERSION%+*}` here instead of `10.4`

Comment (lhw828) [reply apps/budgie-desktop/DETAILS]: It has been corrected.

**Pull request merged**